src/window.rs

```diff
@@ -1522,10 +1522,8 @@ unsafe impl HasRawWindowHandle for Window {
     /// [`Event::Resumed`]: crate::event::Event::Resumed
     /// [`Event::Suspended`]: crate::event::Event::Suspended
     fn raw_window_handle(&self) -> RawWindowHandle {
-        struct Wrapper(RawWindowHandle);
-        unsafe impl Send for Wrapper {}
         self.window
-            .maybe_wait_on_main(|w| Wrapper(w.raw_window_handle()))
```

Comment:
I asked this in IRC, but I'm confused about this part.
How can we return `RawWindowHandle` here if it isn't `Send`? Shouldn't we panic if this isn't the main thread? As far as I can tell on every backend we just return it without a second thought, at least on Web this is invalid.

Reply:
You can do whatever you want with raw-window-handle, since it's just a pointer. However you can access it only in a specific manner most of the time, which is thread safe for winit.